piccolo_api/crud/endpoints.py

```diff
@@ -376,6 +383,24 @@ def _split_params(params: t.Dict[str, t.Any]) -> Params:
                     response.page = page
                 continue
 
+            if key == "__cursor":
+                try:
+                    cursor = str(value)
+                except ValueError:
+                    logger.info(f"Unrecognised __cursor argument - {value}")
+                else:
+                    response.cursor = cursor
+                continue
+
+            if key == "__previous":
+                try:
+                    previous = str(value)
+                except ValueError:
+                    logger.info(f"Unrecognised __previous argument - {value}")
+                else:
+                    response.previous = previous
```

Comment:
I get some type errors for `response.cursor = cursor` and `response.previous = previous` - we just need to add `cursor` and `previous` as attributes to `Params`.

Reply:
@dantownsend Like this:
```
@dataclass
class Params:
    operators: t.Dict[str, t.Type[Operator]] = field(
        default_factory=lambda: defaultdict(lambda: Equal)
    )
    match_types: t.Dict[str, str] = field(
        default_factory=lambda: defaultdict(lambda: MATCH_TYPES[0])
    )
    fields: t.Dict[str, t.Any] = field(default_factory=dict)
    order_by: t.Optional[OrderBy] = None
    include_readable: bool = False
    page: int = 1
    page_size: t.Optional[int] = None
    cursor: str = ""
    previous: str = ""
```
I can add this in next commit.

Reply:
yeah, exactly